packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -508,6 +516,7 @@ class ScaleGestureRecognizer extends OneSequenceGestureRecognizer {
           localFocalPoint: PointerEvent.transformPosition(_lastTransform, _currentFocalPoint),
           rotation: _computeRotationFactor(),
           pointerCount: _pointerQueue.length,
+          delta: _currentFocalPoint - _initialFocalPoint,
```

Comment:
The doc says `delta` will be in the coordinate space of the event receiver but there it looks like these 2 are pre-transformed (global) coordinates (because there's a `localFocalPoint` that has a differenet value)?